runtime/src/bank.rs

```diff
@@ -71,6 +71,7 @@ pub const SECONDS_PER_YEAR: f64 = 365.25 * 24.0 * 60.0 * 60.0;
 pub const MAX_LEADER_SCHEDULE_STAKES: Epoch = 5;
 
 type BankStatusCache = StatusCache<Result<()>>;
+#[frozen_abi(digest = "7dg6BreYxTuxiVz6aLvk3p2Z7GQk2cJqfGvC9h4FAoSj")]
```

Comment:
FYI: @jackcmay When you change any `struct`s and `enum`s indirectly referenced by `InstructionError`, this digest value will need to be updated after checking the abi change details by: https://github.com/solana-labs/solana/pull/10335/files#r433615921

Yeah, this is a bit unintituitive but, this is what is stored in the snapshot and it contains `InstructionError`. :)